crates/iota-light-client/.gitignore

```diff
@@ -1 +1 @@
-checkpoints_dir/*
\ No newline at end of file
+checkpoints*/
```

Comment:
Feels like this could inadvertently ignore something we wouldn't want to, maybe we should should be explicit

Reply:
Making sure you didn't forget about this